src/app/approve/approve.component.ts

```diff
@@ -247,6 +248,17 @@ export class ApproveComponent implements OnInit {
         const groupKeyName = this.transaction.metadata.messagingGroupKeyName
         description = `register group key with name "${groupKeyName}"`
         break
+      case TransactionMetadataDAOCoinLimitOrder:
+        let buyingDAOCoinCreatorPublicKey = this.base58KeyCheck(this.transaction.metadata.buyingDAOCoinCreatorPublicKey);
+        if (buyingDAOCoinCreatorPublicKey == "") {
+          buyingDAOCoinCreatorPublicKey = "DESO";
+        }
```

Comment:
If it's not the empty string, we'll want to add it to the `publicKeys` variable so we can get a username in place of the public key in the description in identity. See line 244 for how it's handled for DAO Coin transfers